modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/portlet/AssetPublisherPortlet.java

```diff
@@ -583,22 +575,23 @@ private String _exportToRSS(
 
 		syndLinks.add(selfSyndLink);
 
-		String feedURL = _getFeedURL(portletRequest);
```

Comment:
Hey @RenanUmV, keep the `String feedURL = _getFeedURL(portletRequest);` statement since the method result is not cacheable and it does not a simple `getter`.